tests/enum.lisp

```diff
@@ -64,12 +66,57 @@
          (eq :true (return-enum2 :true)))
   t)
 
+(defctype numeros-typedef numeros)
+
+(deftest enum.typedef.1
+    (eq (foreign-enum-keyword 'numeros-typedef 1)
+        (foreign-enum-keyword 'numeros 1))
+  t)
+
+(deftest enum.typedef.2
+    (eql (foreign-enum-value 'numeros-typedef :four)
+         (foreign-enum-value 'numeros :four))
+  t)
+
+(defcenum enum-size.int
+  (enum-size-one 1)
+  (enum-size-int #.(1- (expt 2 (1- (* (foreign-type-size :int) 8)))))
+  (enum-size-two 2))
+
+(defcenum enum-size.long
+  (:long #.(1- (expt 2 (1- (* (foreign-type-size :long) 8)))))
+  (:one 1)
+  (:two 2))
+
+#+nil ;; :long-long is often the same size as :long
```

Comment:
because :long and :long-long is the same length on many paltforms. but then the same applies to :int and :long, so i rewrote this test.